WordPress/Classes/ViewRelated/Cells/WPProgressTableViewCell.m

```diff
@@ -36,6 +38,12 @@ - (void)prepareForReuse
     [self.progressView stopAnimating];
 }
 
+- (void)layoutSubviews
+{
+    [super layoutSubviews];
+    self.imageView.frame = CGRectInset(self.imageView.frame, 0, 5);
```

Comment:
Just being nitpicky, could all the values passed to CGRectInset be floats instead of integers for type conformity?